src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -6,7 +6,6 @@
     <AssemblyName>System.Diagnostics.Process</AssemblyName>
     <DefineConstants Condition="'$(TargetGroup)' != 'uap'">$(DefineConstants);FEATURE_REGISTRY</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>$(NoWarn);CS0419;CS1572;CS1573;CS1587;CS1591</NoWarn>
```

Comment:
📝 I generally recommend leaving three warnings disabled across a project:

* CS1573: Parameter 'parameter' has no matching param tag in the XML comment for 'parameter' (but other parameters do)
* CS1591: Missing XML comment for publicly visible type or member 'Type_or_Member'
* CS1712: Type parameter 'type_parameter' has no matching typeparam tag in the XML comment on 'type_or_member' (but other type parameters do)

Reply:
Thanks for the quick review.
As I understand we have CS1573 and CS1591 globally disabled via [Directory.Build.props](https://github.com/dotnet/corefx/blob/master/Directory.Build.props#L482)

Reply:
Thanks!

Reply:
> As I understand we have CS1573 and CS1591 globally disabled via Directory.Build.props

I would like to move away from that, though, with these disabled only for the projects that need it disabled.  We have projects that care a lot about XML comments as the primary source of documentation / IntelliSense / etc., and such warnings should be errors for those projects rather than being suppressed.

Reply:
Should we do it as a part of #33934 ?